src/registrar/forms/application_wizard.py

```diff
@@ -112,11 +111,11 @@ def to_database(self, obj):
             obj.authorizing_official = contact
             obj.save()
 
-    def from_database(self, obj):
+    @classmethod
+    def from_database(cls, obj):
         """Initializes this form's fields with values gotten from `obj`."""
```

Comment:
**suggestion (non-blocking):** It seems like the docstrings for these subclass overrides are copy-pasted from the original method. I think that we could either leave them out if they are the exact same, or use them instead to explain what is specifically different about this subclass's version of the method.

Reply:
They were copy-pasta; I've removed them from all except the base class.